### ISSUE TITLE
Event logging

### ISSUE THREAD

**Issue by dpaiv0** [SwitchbladeBot/switchblade#819]
( Extends #817 )
This is pretty much self explanatory, can be modular and toggleable through the website.

**Comment by dpaiv0** [comment]
Something like @polluxbot or @lorittabot's website
![image](https://user-images.githubusercontent.com/32089154/57111267-4c289b00-6d33-11e9-98de-f86154409e80.png) 
![image](https://user-images.githubusercontent.com/32089154/57111287-72e6d180-6d33-11e9-9f51-0d50cb31d662.png)

**Comment by metehus** [comment]
i wass thinking of custom events like, i set channel #welcome to log joins and leaves maybe with custom message and #moderation-room for bans and kicks and etc, but i think thats too complex ahhdbsjabs

**Comment by pedrofracassi** [comment]
I was thinking of making that customizable per channel, like having it log different things to different channels, or maybe even letting it log the same thing to multiple channels

**Comment by dpaiv0** [comment]
yeah, a per-channel config may be better in this type of cases. but we need some way to filter JUST text channels, and for some reason, categories are also considered as text channels, so we need to find a way to circumvent that

**Comment by metehus** [comment]
cant we just filter if the chann type is text? isnt this very simple?

**Comment by dpaiv0** [comment]
yeah you can, I was super high when I wrote that, sorry

**Comment by metehus** [comment]
yes i get high when im answearing issues on github too
